integration-tests/kubernetes/quarkus-standard-way/src/test/java/io/quarkus/it/kubernetes/KubernetesWithSidecarAndJibTest.java

```diff
@@ -91,6 +91,10 @@ private void assertSidecar(io.fabric8.kubernetes.api.model.PodSpec podSpec) {
                     assertThat(c.getPorts()).singleElement().satisfies(p -> {
                         assertThat(p.getContainerPort()).isEqualTo(3000);
                     });
+                    assertThat(c.getEnv()).singleElement().satisfies(e -> {
```

Comment:
Can we also assert that the env vars are **not** added to the application container?

Reply:
Done.